libraries/AP_Common/missing/cmath

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include_next <cmath>
+
+#if defined(_GLIBCXX_CMATH)
```

Comment:
this is actually wrong... a) it's a "private" guard used by glibcxx... not to be used here; b) we should ifdef with what the build system detected; c) I think this fails if they are not undef'ed by the header, i.e. it fails on !g++ and g++ < 5.3